contracts/SpokePoolV3Periphery.sol

```diff
@@ -706,6 +737,16 @@ contract SpokePoolV3Periphery is SpokePoolV3PeripheryInterface, Lockable, MultiC
         );
     }
 
+    function _paySubmissionFees(
+        address swapToken,
```

Comment:
nit: Do you want to call this `swapToken` since this is also called by the deposit entrypoints? Maybe just `feeToken`?